sqllineage/core/handlers/cte.py

```diff
@@ -23,6 +23,11 @@ def _handle(self, token: Token, holder: SubQueryLineageHolder) -> None:
             cte = [
                 token for token in token.tokens if isinstance(token, cte_token_types)
             ]
+        elif isinstance(token, Parenthesis):
+            # CREATE TABLE tbl1 (col1 VARCHAR) WITH (bucketed_by = ARRAY['col1'], bucket_count = 256).
+            # This syntax is valid for bucketing in Trino and not the CTE
+            self._handle(token.tokens[1], holder)
```

Comment:
Why do we still want to handle token.tokens[1] here?

Reply:
Good catch. Fixed on 8ee1b34

I originally thought the parenthesis should be parsed, but looking at the examples, no longer needed.